tests/web/mock_web_api_server.py

```diff
@@ -102,11 +104,23 @@ def _handle(self):
 
                 if pattern == "html_response":
                     self.send_response(404)
-                    self.set_common_headers()
+                    self.send_header("content-type", "text/html;charset=utf-8")
+                    self.send_header("connection", "close")
+                    self.end_headers()
                     self.wfile.write(self.html_response_body.encode("utf-8"))
                     self.wfile.close()
                     return
 
+                if pattern == "error_html_response":
+                    self.send_response(503)
```

Comment:
Nice, this was also the HTTP status code of the error that we experienced earlier this week. 👌🏻